### PR TITLE
dev: connected to Mongo Atlas

### LINKED ISSUE #48: set up MongoDB
**schema**

list of sessions 
--> session: name 
--> list of users 
-----> user: name, password, email, isHost, list of menu items

list of restaurant menus
--> list of menu items 
-----> menu item: name, price, category

### PR DESCRIPTION
**Pull request by christylo**
restructured folders and connected app to Mongo Atlas db
created models for users, sessions, menus and items
run `npm start` in server directory to connect to Mongo and start server

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose")
+
+const mongoString = "mongodb+srv://m001-student:m001-mongodb-basics@cluster0.dvh8w.mongodb.net/myFirstDatabase?retryWrites=true&w=majority"
```
Comment (kozr) [inline server/db/index.js]: might not be safe to leave this in the public. Anyone could edit the db this way

**Comment by rzyxie** [comment]
FYI i think the server directory might conflict/overlap with some of the changes i made in my draft express PR as well, would those changes fit with what you have going on here? (i've also made a server directory to contain the express files) 

also, i think we can add server/node_modules into the gitignore file so we aren't committing like 1000+ files, that is what we are doing with the node_modules files for the react part of the app if you want to take a look (i have this added in my draft PR already as well)

File: server/db/index.js (hunk 1 above)
Comment (christylo) [reply server/db/index.js]: resolved

**Review by kozr — APPROVED** [review]
Good job! Thanks for setting the database up

**Pull request merged**